js/impress.js

```diff
@@ -364,12 +364,12 @@
             root.appendChild( canvas );
 
             // Set initial styles
-            document.documentElement.style.height = "100%";
+            // document.documentElement.style.height = "100%";
 
-            css( body, {
-                height: "100%",
-                overflow: "hidden"
-            } );
+            // css( body, {
+            //     height: "100%",
+            //     overflow: "scroll"
+            // } );
```

Comment:
Can you explain what you are trying to accomplish here, and why this fixes it?

Reply:
Hello, please delete this PR, I’m so sorry! Wasn’t aware I even did it.
I’m using Impress as part of the page in an LMS, so I needed the scroll to work, but the PR should never been made here like this (I probably used the wrong repository in a hurry)
Sorry again!